Libraries/LibGfx/SkiaUtils.cpp

```diff
@@ -140,4 +143,114 @@ sk_sp<SkImageFilter> to_skia_image_filter(Gfx::Filter const& filter)
+    }
+    if (compositing_and_blending_operator == "hue"sv) {
+        return SkBlender::Mode(SkBlendMode::kHue);
+    }
+    if (compositing_and_blending_operator == "saturation"sv) {
+        return SkBlender::Mode(SkBlendMode::kSaturation);
+    }
+    if (compositing_and_blending_operator == "color"sv) {
+        return SkBlender::Mode(SkBlendMode::kColor);
+    }
+    if (compositing_and_blending_operator == "luminosity"sv) {
+        return SkBlender::Mode(SkBlendMode::kLuminosity);
+    }
+    if (compositing_and_blending_operator == "clear"sv) {
+        return SkBlender::Mode(SkBlendMode::kClear);
+    }
+    if (compositing_and_blending_operator == "copy"sv) {
+        return SkBlender::Mode(SkBlendMode::kSrc);
+    }
+    if (compositing_and_blending_operator == "source-over"sv) {
+        return SkBlender::Mode(SkBlendMode::kSrcOver);
+    }
+    if (compositing_and_blending_operator == "destination-over"sv) {
+        return SkBlender::Mode(SkBlendMode::kDstOver);
+    }
+    if (compositing_and_blending_operator == "source-in"sv) {
+        return SkBlender::Mode(SkBlendMode::kSrcIn);
+    }
+    if (compositing_and_blending_operator == "destination-in"sv) {
+        return SkBlender::Mode(SkBlendMode::kDstIn);
+    }
+    if (compositing_and_blending_operator == "source-out"sv) {
+        return SkBlender::Mode(SkBlendMode::kSrcOut);
+    }
+    if (compositing_and_blending_operator == "destination-out"sv) {
+        return SkBlender::Mode(SkBlendMode::kDstOut);
+    }
+    if (compositing_and_blending_operator == "source-atop"sv) {
+        return SkBlender::Mode(SkBlendMode::kSrcATop);
+    }
+    if (compositing_and_blending_operator == "destination-atop"sv) {
+        return SkBlender::Mode(SkBlendMode::kDstATop);
+    }
+    if (compositing_and_blending_operator == "xor"sv) {
+        return SkBlender::Mode(SkBlendMode::kXor);
+    }
+    if (compositing_and_blending_operator == "lighter"sv) {
+        return SkBlender::Mode(SkBlendMode::kPlus);
+    }
+    if (compositing_and_blending_operator == "plus-darker"sv) {
+        // FIXME: This value is only correctly supported by Safari in combination with Core Graphics.
+        //        It looks like the spec is wrong about the formula. We therefore use the one from
+        //        https://developer.apple.com/documentation/coregraphics/cgblendmode/plusdarker for now.
+        //        However it also does not work.
+        return SkRuntimeEffect::MakeForBlender(SkString(R"(
+            vec4 main(vec4 source, vec4 destination) {
+                return max(vec4(0.0), source + destination - vec4(1.0));
+            }
+        )"))
+            .effect->makeBlender(nullptr);
+    }
+    if (compositing_and_blending_operator == "plus-lighter"sv) {
+        return SkRuntimeEffect::MakeForBlender(SkString(R"(
+            vec4 main(vec4 source, vec4 destination) {
+                return min(vec4(1.0), source + destination);
+            }
```

Comment:
nit: to align with hinted code, you can also switch to
```suggestion
            vec4 main(vec4 source, vec4 destination) {
                return saturate(source + destination);
            }
```